### PR TITLE
Initialize Way Lab EM morphology Tokyo bucket

### PR DESCRIPTION
**Pull request by d33bs**
This PR initializes a new bucket focused on helping facilitate data transfer of EM data from collaborators in Tokyo. It leveraged the [cookiecutter template found here](https://github.com/CU-DBMI/gc-cloud-storage-bucket) with a few modifications. In this case, the collaborators all have Google accounts, so we do not need the key-based functionality from the template.

CC @falquaddoomi, @MikeLippincott

### CODE REVIEW

**Review by gwaybio — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,98 @@
...
+1. Fill in [terraform.tfvars](terraform/state-management/terraform.tfvars) with values that make sense for your initiative (note: these are by default filled in from cookiecutter values).
+1. Terraform __init__: to ensure Terraform is initialized use command `terraform -chdir=terraform/state-management init`.
+1. Terraform __plan__: to plan the work and observe any needs use command `terraform -chdir=terraform/state-management plan` .
+1. Terraform __apply__: to apply the work and create resources use command `terraform -chdir=terraform/state-management apply`
+
+### 🏗️ Bucket Infrastructure
+
+These steps cover how to control the infrastructure found within this repository.
+
+| <span style="text-align:left;float:left;font-weight:normal;">:exclamation:  Please note: after applying the Terraform code with the steps below, a `service-account.json` file is added to your local `/utilities/data-provider` directory which contains sensitive data which may enable access to your cloud resources. __This file should not be checked into source control!__</span>   |
```
Comment (falquaddoomi) [inline README.md]: I saw that you'd removed generation of the `service-account.json` service account credential file (which, by the way, is totally fine if users can interactively log in to get permissions to the bucket...in fact I'd consider it preferable). This is one of a few references to that file that still exist in this repo; I'll comment on the others, too.

File: utilities/data-provider/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Please reference the following directory tree structure for an example of what the path should contain:
+
+```shell
+.
+├── README.md
+├── data
+│   └── <data to be synchronized>
+├── gsutil_sync.bat
+└── service-account.json
+```
```
Comment (falquaddoomi) [inline utilities/data-provider/README.md]: As mentioned in a previous comment, this section still references the nonexistent `service-account.json` file.

You might consider revising these instructions to either tell them to explicitly log in (e.g., via `gcloud auth login`), or that the `gsutil_sync.bat` script will prompt them to log in if you decide to add a login step there.

File: utilities/data-provider/gsutil_sync.bat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+::  presumes gsutil has already been installed and is
+::  available in the path.
+::  see gsutil docs for more information:
+::  https://cloud.google.com/storage/docs/gsutil_install
+
+::  authenticate gcloud for the service account
+::  note: this is the preferred method for authenticating gsutil
+::  see the following for more details:
+::  https://cloud.google.com/storage/docs/gsutil/commands/config#configuring-service-account-credentials
+call gcloud auth activate-service-account --key-file=service-account.json
```
Comment (falquaddoomi) [inline utilities/data-provider/gsutil_sync.bat]: This line still references the nonexistent `service-account.json` file.

Maybe you could invoke `gcloud auth login` here instead, to establish their user credentials? I'm not 100% sure, but I believe that command will detect if the user is already logged in and skip reauthenticating them, unless you pass `--force`.

**Review by falquaddoomi — COMMENTED** [review]
Nice! I assume this is nearly identical to the `gc-cloud-storage-bucket` template, so I didn't get too into reviewing parts I think I've already reviewed before. Let me know if you'd like me to review (or re-review) them in more detail, though.

As you noted in your comment, you're not downloading the service account keyfile any more, so I left a few comments on places where it's still referenced.

File: README.md (hunk 1 above)
Comment (d33bs) [reply README.md]: Thanks, updated!

File: utilities/data-provider/README.md (hunk 2 above)
Comment (d33bs) [reply utilities/data-provider/README.md]: Thank you, updated!

File: utilities/data-provider/gsutil_sync.bat (hunk 3 above)
Comment (d33bs) [reply utilities/data-provider/gsutil_sync.bat]: Thanks, I updated this as well! I found that the `gcloud auth login` always will attempt to login, even if the session is already authenticated for the user. Either way, the command appears to halt until the browser session is ended or the command is cancelled, so I feel good about this working as is (granted someone has browser access). If this doesn't work, they could still use `rclone` as another workaround.

**Comment by d33bs** [comment]
Thank you @gwaybio and @falquaddoomi for your reviews! Made some changes, and merging this in now.

**Pull request merged**